src/yang/ietf-bgp.yang

```diff
@@ -219,6 +219,8 @@ module ietf-bgp {
            graceful restart with the peer";
       }
     }
+
+    uses structure-neighbor-group-error-options;
```

Comment:
I am not a big fan of single use grouping. If this is the only place this grouping is going to be used, I would rather that we inline the grouping.